docs/UserGuide.md

```diff
@@ -60,30 +61,72 @@ You can start from the [Quick Start](#quick-start) section to learn how to obtai
 1. Refer to the [Features](#features) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
-## About this document  <a name="about-this-document"></a>
-This document introduces the features of tCheck. 
+## 3. About <a name="about"></a>
 
-The following are symbols and formatting used in this document:
+### 3.1 Structure of this document
 
-:bulb: 
-Tips are used to describe small features or techniques
-that may come in handy when using tCheck!
+We have structured this User Guide in a way so that you can find what you need easily and quickly. In the next section,
+[Section 3.2 - Reading this document](#reading-this-document), you can find sevaral useful tips on how to read this guide. The following section,
+[Section 4 - Features](#features), documents the four main features in **tCheck**, namely:
 
+   * Sales Tracking
 
-:information_source: 
-Notes describe certain features or behaviour that may
-not be so obvious!
+   * Ingredients Tracking
 
---------------------------------------------------------------------------------------------------------------------
+   * Manpower Management
+
+   * Other general features
+
+### 3.2 Reading this document <a name="reading-this-document"></a>
+
+This section introduces you to some key technical terms, symbols and syntax that are used throughout this guide.
+You may want to familarize yourself with them before moving on to the next section.
+
+#### 3.2.1 Terminology related to the GUI
+
+Figure 2 shows the GUI of **tCheck**, annotated with descriptions for all components of the GUI.
+
+<img src="images/tCheckInfographic.png" width="1428" height="1000" />
```

Comment:
Try to resize again such that the screenshot does not looks squeezed horizontally.